omise/omise.php

```diff
@@ -0,0 +1,44 @@
+<?php
+if (! defined('_PS_VERSION_'))
+    exit();
+
+class Omise extends PaymentModule
+{
+
+    /**
+     * The name that will be display to the user at the back-end.
+     *
+     * @var string
+     */
+    const MODULE_DISPLAY_NAME = 'Omise';
+
+    /**
+     * The name that used to referece in the program.
+     *
+     * @var string
+     */
+    const MODULE_NAME = 'omise';
+
+    /**
+     * The version of the module.
+     *
+     * @var string
+     */
+    const MODULE_VERSION = '1.6.0.0';
+
+    public function __construct()
+    {
+        $this->name = self::MODULE_NAME;
```

Comment:
@guzzilar I disagree. There are no PHP coding standard mention or encourage this style. I found no declaration for this style to be should or must. Including, PrestaShop 1.6 and 1.7 did not use this style also.

To be consistent this style, it give more work to other developer.

For an example, if one of them would be named longer or shorter, all involved variables need to be aligned variables with it.

This style requires human do it manually. I found no any automated tool to do this style.

Although it is the benefit of this style but it is unnecessary in this point.